File: 1/1.cpp
#include <iostream>
using namespace std;

bool is_divisable_5or3(const int n) {
	if (n%3 == 0 or n%5 == 0) return true;
	return false;
}       


int main() {
	int n = 1000;
	int sum = 0;

	for (int i=1; i<n; i++) {
		if (is_divisable_5or3(i)) {
			sum += i;
		}
	}
	cout << sum << endl;
	return 0;
}

File: 1/1.py
def is_divisable_5or3(n: int) -> bool:
	if n%3 == 0 or n%5 == 0:
		return True
	return False


n: int = 1000
s: int = 0

for i in range(1, 1000):
	if is_divisable_5or3(i):
		s += i

print(s)